### DISCUSSION TITLE
[Feature] Add option to also share people/location in partner sharing.

### DISCUSSION THREAD

**Discussion by primetime00** [immich-app/immich#14435] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, my wife and I use Immich to upload photos from our phones.  We also have partner sharing enabled so we can share our libraries.  I am moving from Google Photos to Immich and have everything up and running, but I did notice that I can only see recognized people from my wife's photos when logged into her account, and recognized people from my photos when logged into my account. Would it be possible in the future to add another option in partner sharing that shares this data as well?  We have a large collection of photos, and I'd like to be able to people search without logging into two different users.

Thanks!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#12614
